common/models/user.json

```diff
@@ -148,10 +148,6 @@
       "description": "The challenge last visited by the user",
       "default": ""
     },
-    "currentChallenge": {
```

Comment:
We still need this, because of our legacy database. This will mutate the User Object (old) in a way that we do not want.